pkg/che/tokencache_test.go

```diff
@@ -359,7 +359,7 @@ func keycloackRoute(tls bool) *routev1.Route { //nolint: unparam
 		},
 		Spec: routev1.RouteSpec{
 			Host: fmt.Sprintf("keycloak-codeready-workspaces-operator.%s", test.MemberClusterName),
-			Path: "",
+			Path: "/",
```

Comment:
the test should work without this change, right?
might be a good idea to have the test when path is not defined